### PR TITLE
Multiple small bugfixes

### PR DESCRIPTION
**Pull request by Ch1pStar**
##### Description of change

Multiple small one-line bug fixes
##### Pre-Merge Checklist

- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

File: plugins/text-canvas/src/CanvasTextStyle.js [hunk 1]
```diff
@@ -236,7 +236,7 @@ export default class CanvasTextStyle
         {
             if (k.charAt(0) !== '_') continue;
 
-            Object.defineProperty(k.replace('_', ''), {
+            Object.defineProperties(Object(k.replace('_', '')), {
                 get()
```
Comment (englercj) [inline plugins/text-canvas/src/CanvasTextStyle.js:240]: I think this was supposed to be `Object.defineProperty(this, k.replace, {})`

**Comment by englercj** [comment]
I'll merge when I get back from vacation

File: plugins/text-canvas/src/CanvasTextStyle.js (hunk 1 above)
Comment (Ch1pStar) [reply plugins/text-canvas/src/CanvasTextStyle.js:240]: Yeah, I realised that after submiting it.

**Comment by englercj** [comment]
Fixed the issues mentioned here. Thanks!

**Pull request closed without merging**